src/commands/module/add.ts

```diff
@@ -88,7 +92,9 @@ export default defineCommand({
 
     // Update nuxt.config.ts
     if (!ctx.args.skipConfig) {
-      await updateNuxtConfig(cwd, (config) => {
+      const rootDir = ctx.args.layer ? resolve(cwd, ctx.args.layer) : cwd
```

Comment:
Without considering layer names, isn’t it same as —cwd ?

Reply:
I think setting `--cwd` pointing to the layer dir would affect other places as well, e.g. where `module add` looks for package.json or where it adds a dependency using `nypm` util. Correct me if I'm wrong though.

Reply:
I think what is expected is that the module is added to the closest package.json and nuxt.config files found to the layer. (ie: We need a look-up search for both) 

I think it would probably be best first action to resolve any issues that prevents this. 

—-

In next steps, —layer would be also good idea but i expect that it loads nuxt.config, and tries to match a layer by its name (which used in extends key) basically as a convenient shortcut and also it should implement safety check that we don’t write to readonly (remote or dependency) layers but instead ask to directly add to the root.